### PR TITLE
chore: fix the accounts type

### PR DESCRIPTION
**Pull request by Asalem1**
This was being generated as type: any, this should fix that

### CODE REVIEW

**Review by randycoulman — APPROVED** [review]

File: src/unity/schemas/Account.yml [hunk 1]
```diff
@@ -13,4 +13,13 @@ properties:
...
-required: [id, balance, type]
+  billingContact:
+    $ref: './BillingContact.yml'
+    description: billing contact for the account
+  users:
+    type: array
+    $ref: './Users.yml'
+  marketplaceSubscription:
+    $ref: './MarketplaceSubscription.yml'
+    description: which marketplace, nil if none
```
Comment (Asalem1) [inline src/unity/schemas/Account.yml:24]: added these back as part of the Account type since we only ever reference the Account in the Operator pages

**Review by randycoulman — APPROVED** [review]
I'm OK with this if it's going to make it easier to get the operator UI running, but I'm worried that we're going to have to revert all of these changes when we do multi-account, so I'm wondering if it be easier to tackle that here instead.

**Comment by Asalem1** [comment]
> I'm OK with this if it's going to make it easier to get the operator UI running, but I'm worried that we're going to have to revert all of these changes when we do multi-account, so I'm wondering if it be easier to tackle that here instead.

That's what my concern was when I was making these changes, but truthfully I think it was originally intended that operator-specific types would be integrated to avoid overcrowding these definitions. The things that are most relevant to the operator page are that the account has a `billingContact`, `users`, and `marketplaceSubscription` (and sometimes `organizations`), and that the organizations have related account (linked). I'd be more than happy to drop this PR and have those specific operator types defined as separate entities if you're interested. I'm just not sure if I have enough context on what those definitions would need to contain in order to make each of those relevant to this without tacking on more to the existing schemas

**Comment by randycoulman** [comment]
I think what Iris was doing with this spec was keeping the "common" Account/Org types uncluttered by the needs of the operator UI and then adding in the extra stuff that the operator UI needs.  This change is going the other direction, putting all of the extra stuff into the base type, which will make it difficult in multi-account when we want to return only a small subset of that information.

**Comment by Asalem1** [comment]
> I think what Iris was doing with this spec was keeping the "common" Account/Org types uncluttered by the needs of the operator UI and then adding in the extra stuff that the operator UI needs. This change is going the other direction, putting all of the extra stuff into the base type, which will make it difficult in multi-account when we want to return only a small subset of that information.

Gotcha. So I’ll refactor this to create a new type based off of the common data of a particular type and go from there to create an operator specific type(s)

**Comment by Asalem1** [comment]
@randycoulman I think this should be all set for a rereview now. Let me know if there are any issues with this implementation or if this works

**Review by randycoulman — APPROVED** [review]
I think this looks OK. My two main concerns that appear to be satisfied by this:

- No major changes are required to the already-existing Quartz endpoints that are already in place and working.
- We don’t cause a bunch of re-work when we start multi-account in 2 weeks.

File: contracts/priv/unity.yml [hunk 2]
```diff
@@ -740,33 +740,13 @@ paths:
...
-                        items:
-                          allOf:
-                            - $ref: '#/components/schemas/Organization'
-                            - properties:
-                                account:
-                                  description: the org's account
-                                  $ref: '#/components/schemas/Account'
-                            - required:
-                                - account
+                        $ref: '#/components/schemas/Organizations'
```
[the remark this answers is not in the document]
Comment (Asalem1) [reply contracts/priv/unity.yml:748]: I can definitely take a look

[the remark this answers is not in the document]
Comment (Asalem1) [reply contracts/priv/unity.yml:748]: It looks like it might create a circular dependency with the `operator_orgs_orgID.yml` definition

**Pull request merged**